packages/-ember-data/tests/integration/relationships/nested-relationship-test.js

```diff
@@ -38,112 +37,107 @@ module('integration/relationships/nested_relationships_test - Nested relationshi
     Server loading tests
   */
 
-  test('Sideloaded nested relationships load correctly', function (assert) {
+  test('Sideloaded nested relationships load correctly', async function (assert) {
     let store = this.owner.lookup('service:store');
     let adapter = store.adapterFor('application');
 
     adapter.shouldBackgroundReloadRecord = () => {
       return false;
     };
 
-    run(() => {
-      store.push({
-        data: {
+    store.push({
```

Comment:
I assume the payload is the same here, and github's diff just got confused?

Reply:
Yes